src/sidebar/components/sidebar-content.js

```diff
@@ -252,8 +250,14 @@ function SidebarContentController(
   // Re-fetch annotations when focused group, logged-in user or connected frames
   // change.
   $scope.$watch(
-    () => [groups.focused().id, store.profile().userid, ...store.searchUris()],
-    ([currentGroupId], [prevGroupId]) => {
+    () => [groups.focused(), store.profile().userid, ...store.searchUris()],
+    ([currentGroup], [prevGroup]) => {
+      // If the currentGroup is null, clear the selected annotations and return immediatly.
+      if (!currentGroup) {
+        store.clearSelectedAnnotations();
+        return;
+      }
+
       // FIXME - There is a bug here where the set of displayed annotations can
```

Comment:
I'd really like to fix this bug noted in the comments here. I wonder if we might be able to do that with the clearGroups that I added. Perhaps set the focusGroup to null under that particular circumstance thus ensuring the first load of annotations when the focused group is not set doesn't happen.